trappy/base.py

```diff
@@ -179,6 +179,12 @@ def append_data(self, time, comm, pid, cpu, line, data):
         self.line_array.append(line)
         self.data_array.append(data)
 
+    def conv_to_int(self, value):
+        # Handle false-positives for negative numbers
+        if value.lstrip("-").isdigit():
+            value = int(value)
```

Comment:
I would still check and tolerate a `ValueError` here.

```
try:
  value = int(value)
except:
  pass

There can be other cases where this might fail.